_dev/apps/ui/src/components/product-feed/settings/product-selection/product-selection.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="gs-product-selection">
+    <h3 class="font-weight-600">
+      {{ $t('productFeedSettings.productSelection.methodSynch.title') }}
+    </h3>
+    <div class="container-fluid">
+      <div class="row methods-synch">
+        <div
+          class="col col-12 col-md border-primary-400 p-3"
+          :class="{'border-primary-800': synchSelected === 'synchAllProducts'}"
+        >
+          <b-form-radio
+            v-model="synchSelected"
+            name="customSynchRadio"
+            value="synchAllProducts"
+          >
+            <h3 class="font-weight-700 mb-2">
+              {{ $t('productFeedSettings.productSelection.methodSynch.synchAllProducts') }}
+            </h3>
+            <span class="text-primary-600">
+              {{ $t('productFeedSettings.productSelection.methodSynch.synchAllProductsDesc') }}
+            </span>
+          </b-form-radio>
+        </div>
+        <div
+          class="col col-12 col-md border-primary-400 p-3 ml-md-1 mt-1 mt-md-0"
+          :class="{'border-primary-800': synchSelected === 'synchFilteredProducts'}"
+        >
+          <div>
+            <b-form-radio
+              v-model="synchSelected"
+              name="customSynchRadio"
+              value="synchFilteredProducts"
+            >
+              <h3 class="font-weight-700 mb-2">
+                {{ $t('productFeedSettings.productSelection.methodSynch.synchFilteredProducts') }}
+              </h3>
+              <span class="text-primary-600">
+                {{
+                  $t('productFeedSettings.productSelection.methodSynch.synchFilteredProductsDesc')
+                }}
+              </span>
+            </b-form-radio>
+          </div>
+        </div>
+      </div>
+    </div>
+    <actions-buttons
+      :next-step="nextStep"
+      :previous-step="previousStep"
+      @cancelProductFeedSettingsConfiguration="cancel()"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import ActionsButtons from '@/components/product-feed/settings/commons/actions-buttons.vue';
+import ProductFeedSettingsPages from '@/enums/product-feed/product-feed-settings-pages';
+
+export default {
+  name: 'ProductFeedSettingsProductSelection',
+  components: {
+    ActionsButtons,
+  },
+  data() {
+    return {
+      synchSelected: 'synchAllProducts',
```

Comment:
Can you set your values an enum? This should make the management of your values easier in the long-term.